front/src/applications/stdcm/utils/fomatConflicts.ts

```diff
@@ -0,0 +1,82 @@
+import dayjs from 'dayjs';
+
+import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import type { Conflict } from 'common/api/osrdEditoastApi';
+import type { SuggestedOP } from 'modules/trainschedule/components/ManageTrainSchedule/types';
+
+interface ConflictWithInterval {
+  trainIds?: number[];
+  startTime: string;
+  endTime: string;
+  endDate: string;
+  startDate: string;
+  conflictType: string;
+  waypointBefore: string | null;
+  waypointAfter: string | null;
+}
```

Comment:
We don't use interface

```suggestion
type ConflictWithInterval = {
  trainIds?: number[];
  startTime: string;
  endTime: string;
  endDate: string;
  startDate: string;
  conflictType: string;
  waypointBefore: string | null;
  waypointAfter: string | null;
}
```